src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -2,9 +2,11 @@
 
 Release History
 ===============
+
 2.3.17
 ++++++
 * az aks enable-addons /disable-addons: support case insensitive name
+* clarify that "--output" is ignored for "az aks get-credentials"
```

Comment:
You don't need a history entry for this. It's basically a help-text only change.